components/playlist/browser/resources/components/playlistFolder.tsx

```diff
@@ -114,11 +115,38 @@ function EditActionsContainer ({
   )
 }
 
+function useItemIdFromHash () {
+  const getId = () => window.location.hash.replace('#', '')
+  let [idFromHash, setIdFromHash] = React.useState<string>('')
+  const updateId = () => setIdFromHash(getId())
+
+  React.useEffect(() => {
+    updateId()
```

Comment:
I think `getId` and `updateId` should be defined inside the `useEffect`. Otherwise they might not be the same `updateId` and `getId` which are used in the effect.

In this case, it's fine, because they only access global state, but it's more obvious things will work correctly if they're in the effect :smile: